core/src/main/java/com/datastax/oss/driver/internal/core/metadata/schema/parsing/DataTypeCqlNameParser.java

```diff
@@ -110,25 +122,17 @@ private DataType parse(
     }
 
     if (type.equalsIgnoreCase("tuple")) {
-      List<String> rawTypes = parser.parseTypeParameters();
+      if (parameters.isEmpty()) {
+        throw new IllegalArgumentException("Expecting at list one parameter for tuple, got none");
```

Comment:
Shouldn't technically be possible since server won't allow it right?  Was able to exercise it in a unit test though so seems like a good idea :+1:.  I noticed `list<>,set<>,map<>, map<only1>` all throw similar errors too which is good.